stdlib/InteractiveUtils/test/runtests.jl

```diff
@@ -547,9 +547,9 @@ if Sys.ARCH === :x86_64 || occursin(ix86, string(Sys.ARCH))
     output = replace(String(take!(buf)), r"#[^\r\n]+" => "")
     @test !occursin(rgx, output)
 
-    code_native(buf, linear_foo, ())
-    output = String(take!(buf))
-    @test occursin(rgx, output)
+    code_native(buf, linear_foo, (), debuginfo = :none)
```

Comment:
Should this also specify `syntax=:intel` not to rely on the default value?

Reply:
The test right before this one does that. This test seems to specifically check the default.